src/containers/Add/index.jsx

```diff
@@ -21,69 +21,71 @@ class Add extends React.Component {
     errorText: '',
   }
   handleInput = e => {
-    this.setState({ text: e.target.value })
+    this.setState({
+      text: e.target.value,
+    })
   }
   handleSubmit = e => {
     const { time, text } = this.state
-    nervos.appchain
+    appchain.base
       .getBlockNumber()
       .then(current => {
         const tx = {
           ...transaction,
-          from: nervos.appchain.defaultAccount,
+          from: window.neuron.getAccount(), 
```

Comment:
is `window.neuron.getAccount()` synchronous?

Reply:
Yes，it isn't a http request.